Exceptions/src/com/company/Main.java

```diff
@@ -0,0 +1,19 @@
+package com.company;
+
+public class Main {
+
+    public static void main(String[] args) {
+        try {
+            Divider.divDouble();
+        }
+        catch (Exception exception){
+            System.out.println(exception + " : " + exception.getMessage());
+        }
+
+        try {
+            IntNum.validateNumber();
+        } catch (Exception exception){
```

Comment:
catch particular exceptions, not general one

Reply:
fixed